console/command/Test.php

```diff
@@ -67,6 +67,13 @@ class Test extends \lithium\console\Command {
 	public $plain = false;
 
 	/**
+	 * Shows text output with color or without
+	 *
+	 * @var boolean
+	 */
+	public $noColor = false;
```

Comment:
No problem. Its strange, and perhaps should be looked into. but for now, duplicating a property is not an issue I don't think.